package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Sun Nov 17 23:21:31 UTC 2019 - Knut Anderssen <kanderssen@suse.com>
+
+- Network: dropped support for 'bne', 'escon', 'fddi', 'tr',
+  'myrinet', 'xp' and 'usb' netcards (jsc#SLE-7753)
```

Comment:
well I'd simply wrote "for obsolete network device types" ... but that is only NP